src/components/profiles-modals/ImportProfileModal.vue

```diff
@@ -274,7 +274,7 @@ export default class ImportProfileModal extends mixins(ProfilesMixin) {
 </script>
 
 <template>
-    <ModalCard v-if="activeStep === 'FILE_CODE_SELECTION'" key="FILE_CODE_SELECTION" :is-active="isOpen" @close-modal="closeModal">
+    <ModalCard id="import-profile-from-file-or-code-modal" v-if="activeStep === 'FILE_CODE_SELECTION'" key="FILE_CODE_SELECTION" :is-active="isOpen" @close-modal="closeModal">
```

Comment:
Nitpick: wouldn't have minded if all the modals in this file would've used the same `import-profile` or `profile-import` prefix consistently.